faq.md

```diff
@@ -1,112 +1,110 @@
 
-langage étant à remplacer en fonction : cpp pour coloration C/C++, lua pour du Lua, csharp pour du C#, pony pour du Pony, etc. 
+Précision concernant les bots discords : toute demande d'aide concernant de près ou de loin le développement, l'hébergement ou le maintient des bots discord **doit** se faire dans le salon #bots-discord **même si** la question ne concerne pas immédiatement le bot.
 
-Par exemple : 
-```cpp
-std::cout << "Hello world !" << std::endl;
-```
+Précision concernant les bases de données : la section #database concerne principalement les questions relatives aux schémas et aux requêtes. Les questions concernant les bibliothèques spécifiques à leur langages de programmation n'y ont pas leur place.
 
-Qui donnera :
-std::cout << "Hello world !" << std::endl;
+**Sécurité informatique**
 
-/!\ Attention, le caractère ` (accent grave) utilisé ici est différent du ' (guillemet), il peut s'obtenir par la combinaison Alt Gr + 7 sur un clavier azerty.
+Parce que sur ce serveur il y a aussi beaucoup de novices en informatique, nous prenons avec des pincettes tout ce qui touche à la sécurité informatique. Ce sujet n'est abordé *que* lorsqu'il s'agit de renforcer la protection d'un système ou bien de corriger des failles évidentes (exemple: injection sql).
 
-### "J'ai besoin d'aide, venez en MP" est une mauvaise idée (!mp)
-Il est inutile, voire même contre-productif, de demander aux gens de venir en privé pour vous aider. Le serveur dispose d’un grand nombre de canaux sur pratiquement toutes les thématiques du développement, il y en aura forcément un pour votre question.
+Il n'y a personne qui est à la fois habitué, experte reconnue du domaine et qui voudrait devenir ambassadeur d'un canal sécurité. De ce fait nous ne pouvons pas garantir que les discussions concernant le *pentesting*, le *reverse engineering* et d'autres formes de crack/hack se déroule dans un cadre déontologique. Ces discussions ont interdites sur NaN.
 
-De plus, en posant votre question publiquement, l’intelligence collective vous profitera, et vous serez assuré de ne pas recevoir une réponse inadéquate, ou fausse. De ce fait, les demandes d’aides en privé sont interdites sur ce serveur.
+**Web-scrapping**
+
+Le scrapping de données est la pratique de programmatiquement télécharger des données depuis un site. Il est possible de scrapper en forgeant des requêtes HTTP ou bien en automatisant un navigateur web.
+
+Le scrapping n'est pas correctement couvert par un cadre légal. Sur NaN, nous n'autorisons le web-scrapping que lorsque celui-ci peut s'effectuer sans automatisation d'un navigateur (sans selenium) et dans le respect stricte des conditions d'utilisation du site en question. En d'autres termes, lorsqu'un site propose une API et que vous voulez intéragir avec ce site: vous *devez* utiliser cette API.
```

Comment:
> "sans automatisation d'un navigateur (sans selenium)"

Cette parti n'est pas utile et allourdis le paragraphe et ça compréhension, quid des lib classique http par exemple, le paragraphe peu être simplifier en NaN autorise s'il y a un access au donnée ou une autorisation explicite du propriétaire des données (CGU, OpenData, API ...)

Reply:
Je ne suis pas d'accord, le scraping n'est pas illégal. Il est utile dans de nombreux cas et les géants du web comme Facebook ou LinkedIn ont perdu leurs procès pour contrer le scraping.

Il peut uniquement être considéré comme illégal et hors CGU lorsqu'il est nécessaire de s'authentifier comme pour scraper les profils Facebook de ses amis, par exemple.

C'est beaucoup moins borderline que d'autres sujets et ce n'est pas pertinent de l'interdire. 

https://parsers.me/us-court-fully-legalized-website-scraping-and-technically-prohibited-it/